### DISCUSSION TITLE
Unnoficial Arduino UNO (elegoo) not detected after burning

### DISCUSSION THREAD

**Discussion by MininuxDev** [NicoHood/HoodLoader2#97] [Q&A] [answered]
Hi, i burnt the 16u2 of my Elegoo UNO using the Installation sketch method (2.0.5 version of hoodloader). The led blinked slowly, then stopped, and then blinked fast so i think the installation was successful. I tried several times, even with the ISCP wires flipped 180° (the installation failed this time).
The problem is that my pc no longer detects my board after plugging it in. On linux (edit: archlinux), dmesg shows nothing and on windows i can't see it either. So i can't upload any new sketch or revert the 16u2.
I saw that it's possible to burn the 16u2 with another Arduino but i don't have one (and buying an arduino to fix an arduino is kinda weird...).
Is there anything that can be done without buying another board ?

Thanks in advance

**Comment by NicoHood** [comment]
That sounds weird. Can you make the led blink fast again? That is the minimum requirement.

I did a quick research on the Elegoo brand you mentioned, but I could not find any schematic of the board. Maybe they have a custom layout with custom bootloader. But that should normally not be the case. Is it possible that you have an 8u2 instead of a 16u2? But that should also not be able to flash, since the programming sketch should refuse to burn the sketch then. I really have no idea why this happens.

**Comment by MininuxDev** [comment]
Yes I could make it blink fast multiple times (the only time it did not work is with the connectors flipped)
And i'm almost sure it's not a 8u2, it's written in very small 16u2 on the chip 
Also is it normal that the 16u2 gets hot when it gets flashed ? touching it after a few attempts was almost a bit painful 

I'll probably try to reflash the board with the gpio pins of a raspberry pi (I doubt it works it looks complicated) or borrow an arduino from a friend, thanks for your answer !

**Reply by NicoHood** [reply]
If it gets hot, you are possibly shortening a wire. Or you already did and it is already broken. Check the wiring!

I would not recommending to use a raspi, as it can kill your raspi, if you are doing the wrong wiring. Consider buying another arduino. Since you have a cheap china clone, they should not cost more than 3$ if you buy a nano for example. Always handy to have one.

**Comment by MininuxDev** [comment]
Yeah for the raspberry pi i saw the arduino can output 5V which is too much for the rpi, so it needs additional stuff
i searched for elegoo clones of nano but they seem to be sold out :( if i can't repair my current board with the uno clone of my friend i'll probably buy a true nano

**Comment by MininuxDev** [comment]
Update: I managed to upload new sketches to the arduino with my raspberry pi through the ISCP pins, so i reuploaded the installation sketch but this time i tried setting the HEXFILE to the other bootloaders
 With hexfiles 201 and 203 (HEXFILE_Arduino_COMBINED_dfu_usbserial_atmega16u2_Uno_Rev3_hex and Full) my computer still doesn't detect it. With the 302 (HEXFILE_BootloaderDFU_atmega16u2_hex), the installation is successful but then the TX and RX leds are permanently alternating. My computer detects it when i plug it in (as a LUFA DFU) but i couldn't upload sketches.

Maybe Elegoo built a custom firmware in the 16u2 and only that one works ? or my 16u2 is dead but since it is detected with DFU it would be weird...

**Reply by NicoHood** [reply]
I guess you need to get a schematic of the board or the bootloader code.

**Comment by MininuxDev** [comment, marked as answer]
ok i found a solution, instead of flashing the installation sketch and then running it, i directly connected my raspberry pi to the 16u2 ISCP and burnt [this hexfile](https://github.com/jj1bdx/arduino-atmega16u2/blob/master/hexfiles/Genuino-COMBINED-dfu-usbserial-atmega16u2-Uno-R3.hex) and now it works again... Then i tried to flash HoodLoader2 with the same method and it works again... Still weird since the installation sketch was supposedly successful
In any case thanks for your help !

**Reply by NicoHood** [reply]
Great to hear that!
